.github/maintainers_guide.md

```diff
@@ -22,9 +22,9 @@ and to allow lerna to link them to each other.
 ### Testing
 
 This project has tests for individual packages inside of each's respective `test` directory. It also has
-integration tests in the `integration-tests` directory at the root. You can run the entire test suite using the npm
-script `npm test` at the top level. This script will use Lerna to invoke tests in each package and the integration
-tests.
+integration tests in the `integration-tests` directory under the root `support` directory. You can run the entire test
```

Comment:
i'd like to understand the motivation behind moving the `integration-tests` into the `support` directory.

from my point of view, the `support` directory communicates that the contents are there to support the repository itself. so it contains tools and scripts and assets needed to support the repo. i don't think integration tests fit that description, they are meant to be updated with the codebase as it changes. in this view, they are a part of the product rather than support.

Reply:
> the `support` directory communicates that the contents are there to support the repository itself.

I think integration tests *do* fit that description. It might just be my perspective: tests are a tool that help ensure that the codebase--the meat of the repo--are working as expected. I'd even go as far as saying it feels *odd* to me if `integration-tests/` were top-level alongside `support/` (as opposed to inside `support/`).

I think the notion that tools that support the repo remain constant while the codebase changes is flawed. The tools that support the repo also will need to grow as the codebase changes, albeit at different rates.

Reply:
thanks for clarifying and supporting your case! i'm comfortable with your definitions now. let's keep it as you proposed 👍